cartography/intel/aws/ec2/volumes.py

```diff
@@ -26,90 +28,76 @@ def get_volumes(boto3_session: boto3.session.Session, region: str) -> List[Dict]
 
 
 def transform_volumes(volumes: List[Dict[str, Any]], region: str, current_aws_account_id: str) -> List[Dict[str, Any]]:
+    result = []
     for volume in volumes:
-        volume['VolumeArn'] = f"arn:aws:ec2:{region}:{current_aws_account_id}:volume/{volume['VolumeId']}"
-        volume['CreateTime'] = str(volume['CreateTime'])
-    return volumes
+        attachments = volume.get('Attachments', [])
+        active_attachments = [a for a in attachments if a['State'] == 'attached']
+
+        volume_id = volume['VolumeId']
+        raw_vol = ({
+            'Arn': build_arn('ec2', current_aws_account_id, 'volume', volume_id, region),
+            'AvailabilityZone': volume['AvailabilityZone'],
+            'CreateTime': volume['CreateTime'],
+            'Encrypted': volume['Encrypted'],
+            'Size': volume['Size'],
+            'State': volume['State'],
+            'OutpostArn': volume['OutpostArn'],
+            'SnapshotId': volume['SnapshotId'],
+            'Iops': volume['Iops'],
+            'FastRestored': volume['FastRestored'],
+            'MultiAttachEnabled': volume['MultiAttachEnabled'],
+            'VolumeType': volume['VolumeType'],
+            'VolumeId': volume_id,
+            'KmsKeyId': volume['KmsKeyId'],
+        })
```

Comment:
No need for parenthesis
```suggestion
        raw_vol = {
            'Arn': build_arn('ec2', current_aws_account_id, 'volume', volume_id, region),
            'AvailabilityZone': volume['AvailabilityZone'],
            'CreateTime': volume['CreateTime'],
            'Encrypted': volume['Encrypted'],
            'Size': volume['Size'],
            'State': volume['State'],
            'OutpostArn': volume['OutpostArn'],
            'SnapshotId': volume['SnapshotId'],
            'Iops': volume['Iops'],
            'FastRestored': volume['FastRestored'],
            'MultiAttachEnabled': volume['MultiAttachEnabled'],
            'VolumeType': volume['VolumeType'],
            'VolumeId': volume_id,
            'KmsKeyId': volume['KmsKeyId'],
        }
```